packages/lesswrong/components/votes/lwReactions/ReactionsAndLikesVote.tsx

```diff
@@ -127,7 +132,7 @@ const ReactionsAndLikesVote  = ({
   
 
   return <div className={className}>
-    <LWTooltip title={voteScoreTooltip}>
+    <LWTooltip title={voteScoreTooltip} disabled={!!isMobile()}>
```

Comment:
isMobile docstring:
```
/**
 * Returns whether this is a mobile device (according to heuristics in the
 * bowser library). Only usable on the client. Do NOT use this inside a
 * component function outside of an event handler, since that will create
 * an SSR mismatch. If you're thinking of using this to change layout/
 * presentation, this is probably not what you want; use CSS breakpoints
 * instead.
 */
```

I suspect that using it this way will in fact cause an SSR mismatch... since `isMobile` should only be used inside of event handlers, we probably need to refactor `useHover` to accept something like a new optional `disableOnMobile` prop, which checks `isMobile` inside of `handleMouseOver` (and then pass that through `LWTooltip`).